exec/cnex/global.c

```diff
@@ -731,9 +769,185 @@ void dictionary__keys(TExecutor *exec)
 
 
 void number__toString(TExecutor *exec)
+{
+    TString *s = cell_toString(top(exec->stack)); pop(exec->stack);
+    push(exec->stack, cell_fromString(s));
+    string_freeString(s);
+}
+
+
+
+
+void object__getArray(TExecutor *exec)
+{
+    Cell *a = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (a->type != cArray) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to Array", BID_ZERO);
+        return;
+    }
+    push(exec->stack, a);
+}
+
+void object__getBoolean(TExecutor *exec)
+{
+    Cell *b = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (b->type != cBoolean) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to Boolean", BID_ZERO);
+        return;
+    }
+    push(exec->stack, b);
+}
+
+void object__getBytes(TExecutor *exec)
+{
+    Cell *b = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (b->type != cBytes) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to Bytes", BID_ZERO);
+        return;
+    }
+    push(exec->stack, b);
+}
+
+void object__getDictionary(TExecutor *exec)
+{
+    Cell *d = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (d->type != cDictionary) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to Dictionary", BID_ZERO);
+        return;
+    }
+    push(exec->stack, d);
+}
+
+void object__getNumber(TExecutor *exec)
+{
+    Cell *v = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (v->type != cNumber) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to Number", BID_ZERO);
+        return;
+    }
+    push(exec->stack, v);
+}
+
+void object__getString(TExecutor *exec)
+{
+    Cell *s = cell_fromCell(top(exec->stack)->object->cell); pop(exec->stack);
+    if (s->type != cString) {
+        exec_rtl_raiseException(exec, "DynamicConversionException", "to String", BID_ZERO);
+        return;
+    }
+    push(exec->stack, s);
+}
+
+// ToDo: Implement exception handling in object_make() routines.
```

Comment:
What exactly is the exception handling that needs to be implemented here?

Reply:
Removed incorrect code comments.